dali/operators/image/color/brightness_contrast.h

```diff
@@ -91,9 +91,23 @@ class BrightnessContrastOp : public Operator<Backend> {
 
   void AcquireArguments(const workspace_t<Backend> &ws) {
     auto curr_batch_size = ws.GetInputBatchSize(0);
-    this->GetPerSampleArgument(brightness_, "brightness", ws, curr_batch_size);
-    this->GetPerSampleArgument(brightness_shift_, "brightness_shift", ws, curr_batch_size);
-    this->GetPerSampleArgument(contrast_, "contrast", ws, curr_batch_size);
+    if (this->spec_.ArgumentDefined("brightness")) {
```

Comment:
perhaps out of scope, but consider using the ArgValue utility. The code is much simpler to write this way.